File: crs-transformation-adapter-test/src/test/java/com/programmerare/com/programmerare/testData/TestResult.java
package com.programmerare.com.programmerare.testData;

import com.programmerare.crsTransformations.CrsTransformationFacade;

import java.util.List;

public class TestResult {
    // TODO: getters instead of public fields below ...
    public final CrsTransformationFacade facade;
    public final long totalNumberOfSecondsForAllTransformations;
    public final List<TestResultItem> testResultItems;

    TestResult(
        CrsTransformationFacade facade,
        long totalNumberOfSecondsForAllTransformations,
        List<TestResultItem> testResultItems
    ) {
        this.facade = facade;
        this.totalNumberOfSecondsForAllTransformations = totalNumberOfSecondsForAllTransformations;
        this.testResultItems = testResultItems;
    }
}

File: Programmerare.CrsTransformations.Test/TestData/TestResult.cs
using System.Collections.Generic;

namespace Programmerare.CrsTransformations.TestData
{

public class TestResult {
    // TODO change the public fields to properties  
    public ICrsTransformationAdapter adapter;
    public long totalNumberOfSecondsForAllTransformations;
    public IList<TestResultItem> testResultItems;

    public TestResult(
            ICrsTransformationAdapter adapter,
            long totalNumberOfSecondsForAllTransformations,
            IList<TestResultItem> testResultItems
    ) {
        this.adapter = adapter;
        this.totalNumberOfSecondsForAllTransformations = totalNumberOfSecondsForAllTransformations;
        this.testResultItems = testResultItems;
    }
}
}